tools/depends/target/libgpg-error/Makefile

```diff
@@ -27,6 +27,18 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	cd $(PLATFORM); $(CONFIGURE)
 
 $(LIBDYLIB): $(PLATFORM)
+ifeq ($(OS),osx)
+	$(MAKE) -C $(PLATFORM)/src gen-posix-lock-obj
+	$(PLATFORM)/src/gen-posix-lock-obj > $(PLATFORM)/src/syscfg/tmp.h
+	mv $(PLATFORM)/src/syscfg/tmp.h $(PLATFORM)/src/syscfg/$($(AWK) 'NR==1 {print $2}' $(PLATFORM)/src/syscfg/tmp.h)
```

Comment:
I'm sorry, I though $(AWK) was defined in depends...
this is how it works:
>mv $(PLATFORM)/src/syscfg/tmp.h $(PLATFORM)/src/syscfg/$$(awk 'NR==1 {print $$2}' $(PLATFORM)/src/syscfg/tmp.h)

you really need the $$